index.html

```diff
@@ -395,12 +394,10 @@ <h2>The <dfn>IdleDeadline</dfn> interface</h2>
       <ol>
         <li>Let <var>now</var> be a {{DOMHighResTimeStamp}} representing
         <a>current high resolution time</a> in milliseconds.</li>
-        <li>Let <var>deadline</var> be the <a>time</a> associated with the
-        <a>IdleDeadline</a> object.</li>
+        <li>Let <var>deadline</var> be the result of calling the <a>get deadline time</a>
+        algorithm associated with the <a>IdleDeadline</a> object.</li>
```

Comment:
Maybe better:

```suggestion
        <li>Let <var>deadline</var> be the result of calling {{IdleDeadline}}'s <a>get deadline time</a> algorithm.</li>
```

However, can we get {{IdleDeadline}} into a variable? it should really be:

```
       <li>Let |deadline| be the result of calling |idleDeadline|'s [=get deadline time=] algorithm.</li>
```